FirebaseVertexAI/Sources/GenerativeModel.swift

```diff
@@ -104,7 +104,7 @@ public final class GenerativeModel {
   /// [zero-shot](https://developers.google.com/machine-learning/glossary/generative#zero-shot-prompting)
   /// or "direct" prompts. For
   /// [few-shot](https://developers.google.com/machine-learning/glossary/generative#few-shot-prompting)
-  /// prompts, see ``generateContent(_:)-58rm0``.
+  /// prompts, see ``generateContent(_:)``.
```

Comment:
nit: How about `generateContent(_ content: @autoclosure () throws -> [ModelContent])` to differentiate the two overloads and a single set of backticks since it's not linkable?

Reply:
Done.